hack/generate-standard-manifests.sh

```diff
@@ -26,7 +26,8 @@ Generate standard YAML manifests for Antrea using Helm and writes them to output
         --out <DIR>                   Output directory for generated manifests
         --help, -h                    Print this message and exit
 
-In 'release' mode, environment variables IMG_NAME and IMG_TAG must be set.
+In 'release' mode, either environement variable IMG_NAME(for using unified image) or AGENT_IMAGE and
```

Comment:
I don't think we need to support the "unified image" case in this script. For all generated manifests, let's use the separate agent / controller images. We will still publish the unified image to the Docker registry for backwards compatibility (at least for a while), but that's about it.